### PR TITLE
correct doc for DF.min

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 1]
```diff
@@ -914,30 +914,67 @@ def max_overload(df, axis=None, skipna=None, level=None, numeric_only=None):
...
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
 
-       .. only:: developer
+    Pandas API: pandas.DataFrame.min
 
-           Test: python -m sdc.runtests -k sdc.tests.test_dataframe.TestDataFrame.test_min*
+    Limitations
+    -----------
+    Parameters ``axis``, ``skipna``, ``level`` and ``numeric_only`` are unsupported.
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: Actually parameter `skipna` is supported.

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 2]
```diff
@@ -898,30 +898,57 @@ def max_overload(df, axis=None, skipna=None, level=None, numeric_only=None):
...
+        :ref:`Series.max <pandas.Series.max>`
+            Return the maximum.
+        :ref:`Series.idxmin <pandas.Series.idxmin>`
+            Return the index of the minimum.
+        :ref:`Series.idxmax <pandas.Series.idxmax>`
+            Return the index of the maximum.
+        :ref:`DataFrame.sum <pandas.DataFrame.sum>`
+            Return the sum over the requested axis.
+        :ref:`DataFrame.min <pandas.DataFrame.min>`
+            Return the minimum over the requested axis.
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: Is it ok to have link to itself?

Comment (Rubtsowa) [reply sdc/datatypes/hpat_pandas_dataframe_functions.py]: Yes, pandas have in some methods

Comment (Rubtsowa) [reply sdc/datatypes/hpat_pandas_dataframe_functions.py]: @densmirn  https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.max.html

**Pull request merged**